FifthLesson/VectorProgram/VectorProgram/ProgramRun.cs

```diff
@@ -13,87 +9,132 @@ class ProgramRun
-            Console.WriteLine($"{vectorSecond.ToString()} * {numericInput.ToString()} = {(vectorSecond * numericInput).ToString()}");
+            var numericMultiplier = GetUserNumericMultiplier();
+            WriteLine("Multiplication of vectors by number:");
+
+            WriteLine($"{first.ToString()} * {numericMultiplier.ToString()} = {(first * numericMultiplier).ToString()}");
+            WriteLine($"{second.ToString()} * {numericMultiplier.ToString()} = {(second * numericMultiplier).ToString()}\n");
 
-            Console.WriteLine($"{numericInput.ToString()} * {vectorFirst.ToString()} = {(numericInput * vectorFirst).ToString()}");
-            Console.WriteLine($"{numericInput.ToString()} * {vectorSecond.ToString()} = {(numericInput * vectorSecond).ToString()}");
+            WriteLine($"{numericMultiplier.ToString()} * {first.ToString()} = {(numericMultiplier * first).ToString()}");
+            WriteLine($"{numericMultiplier.ToString()} * {second.ToString()} = {(numericMultiplier * second).ToString()}");
         }
 
-        private static void ShowVectorsAndActions(Vector vectorFirst, Vector vectorSecond)
+        private void ShowActionsWithVectors(Vector first, Vector second)
         {
-            Console.WriteLine("Your vectors:");
-            Console.WriteLine($"1 vector: {vectorFirst.ToString()}");
-            Console.WriteLine($"2 vector: {vectorSecond.ToString()}");
+            WriteLine("Actions with vectors:");
+            WriteLine($"{first.ToString()} + {second.ToString()} = {(first + second).ToString()}");
+            WriteLine($"{first.ToString()} - {second.ToString()} = {(first - second).ToString()}");
+            WriteLine($"{first.ToString()} * {second.ToString()} = {(first * second).ToString()}");
+        }
 
-            Console.WriteLine("Actions with vectors:");
-            Console.WriteLine($"{vectorFirst.ToString()} + {vectorSecond.ToString()} = {(vectorFirst + vectorSecond).ToString()}");
-            Console.WriteLine($"{vectorFirst.ToString()} - {vectorSecond.ToString()} = {(vectorFirst - vectorSecond).ToString()}");
-            Console.WriteLine($"{vectorFirst.ToString()} * {vectorSecond.ToString()} = {(vectorFirst * vectorSecond).ToString()}");
+        private void ShowVectors(Vector first, Vector second)
+        {
+            WriteLine("Your vectors:");
+            WriteLine($"1 vector: {first.ToString()}");
+            WriteLine($"2 vector: {second.ToString()}");
         }
 
-        private Vector EnterUserData(string orderByVectors)
+        private void ShowVectorMultiplication(Vector first, Vector second)
         {
-            Console.WriteLine($"Enter the coordinates of the {orderByVectors} three-dimensional vector separated by a space.");
-            var userInput = ReadInputCoordsAndVerificate();
+            var VectorMultiplication = Vector.VectorMultiplicate(first, second);
```

Comment:
Please, use camelCase style for local variables.

Reply:
Correct